modules/fluid_properties/src/userobjects/SodiumProperties.C

```diff
@@ -18,9 +18,17 @@ SodiumProperties::validParams()
 {
   InputParameters params = FluidProperties::validParams();
   params.addClassDescription("Fluid properties for sodium");
+  params.addParam<Real>(
+      "thermal_conductivity",
+      "Optional value for thermal conductivity that overrides interal calculations");
+  params.addParam<Real>("specific_heat",
+                        "Optional value for specific heat that overrides interal calculations");
```

Comment:
If you want this object to be consistent with the other fluid properties classes, then use `k` for thermal conductivity and `cp` for specific heat.

NOTE: this is just a comment, not a request for change.